### PR TITLE
Fix/page component ng lifecycle hooks

### PR DESCRIPTION
**Pull request by jonasjensen-brolstark**
I have moved all the initialization of elements from  `ngAfterContentInit` to `ngAfterContentChecked` so changes are added after each change detection cycle.

We saw an issue with the old way, if our page title or action had an `ngIf` on it. So i have added `ngIf` to the advanced page example, so there is a visual way to test this.

As a bonus i fixed the page showcase page, as it was f'ed up and missing an example..

### CODE REVIEW

File: src/app/examples/page-example/advanced/page-advanced-example.component.ts [hunk 1]
```diff
@@ -27,14 +27,14 @@ const config = {
   
   <!-- Fixed Page Actions -->
   <kirby-page-actions *kirbyPageActions="{fixed: true}">
-    <button kirby-button (click)="onMoreSelect()">
+    <button *ngIf="true" kirby-button (click)="onMoreSelect()">
```
Comment (jakobe) [inline src/app/examples/page-example/advanced/page-advanced-example.component.ts]: Please search and replace this from the template example:
```
  static readonly template = config.template
    .replace(' defaultBackHref="/"', '')
    .replace(' [innerHTML]="content">', '>...')
    .replace(' *ngIf="true"', '');
```

File: src/kirby/components/page/page.component.ts [hunk 2]
```diff
@@ -15,6 +14,7 @@ import {
   AfterContentInit,
```
Comment (jakobe) [inline src/kirby/components/page/page.component.ts]: :fire: Remove unused import
```suggestion
```

File: src/kirby/components/page/page.component.ts [hunk 3]
```diff
@@ -192,13 +192,10 @@ export class PageComponent implements OnInit, OnDestroy, AfterContentInit, After
     });
   }
 
-  ngAfterContentInit(): void {
+  ngAfterContentChecked(): void {
```
Comment (jakobe) [inline src/kirby/components/page/page.component.ts]: Does this have a performance impact? ⚡️

File: src/kirby/components/page/page.component.ts (hunk 2 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/page/page.component.ts]: It is not unused. Is still used on `PageActionsComponent`.

File: src/kirby/components/page/page.component.ts (hunk 3 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/page/page.component.ts]: Not as i see it. The most expensive thing we do, is creating and attaching the `IntersectionObserver`. But this is safeguarded so it only happens once. 
The rest is simple DOM traversal.

There is of course a danger, that future changes will introduce performance issues here.

**Pull request merged**